pupil-spa/src/app/header/header.component.html

```diff
@@ -1,4 +1,4 @@
-<header role="banner" id="global-header" class="with-proposition">
+<header role="banner" id="global-header" class="with-proposition speech-hidden">
```

Comment:
There is already a perfectly good *attribute* for hiding element from speech-readers: 'aria-hidden'.